app/src/docs/content/core/introToCore.mdx

```diff
@@ -84,9 +84,9 @@ All of the main resources types created and used within Core can be shared with
   figCaption="Share button in canvas editor"
 />
 
-When you share a resource, you can share it either privately or publicly. A private share means the link is only accessible to the users whose email address it has been shared with. A public share means anyone with the link — and importantly, anyone using the public API — can access your resource. 
+When you share a resource, you can share it either privately or publicly. A private share means the link is only accessible to the users whose Ethereum address it has been shared with. A public share means anyone with the link — and importantly, anyone using the public API — can access your resource. 
 
-To share any resource, click share, then paste in the email address of the person you want to share it with, and click +. Select the access level you want to give them (view only, can edit or can share) and hit Save to confirm.
+To share any resource, click share, then paste in the Ethereum address of the person you want to share it with, and click +. Select the access level you want to give them (view only, can edit or can share) and hit Save to confirm.
```

Comment:
`..., click share, then paste...` -> `..., click share and paste...`?

Reply:
Permissions are now more fine-grained than just "view only, can edit or can share" if we want to elaborate.

Reply:
I'll tidy up a little but we need a bigger pass, solely for the new sharing sidebar. I think Matt started on that a while back but I didn't followup on it.